jcli/src/jcli_app/certificate/new_vote_plan.rs

```diff
@@ -0,0 +1,36 @@
+use crate::jcli_app::certificate::{write_cert, Error};
+use chain_impl_mockchain::block::BlockDate;
+use chain_impl_mockchain::certificate::{Certificate, Proposals, VotePlan};
+use std::ops::Deref;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+pub struct VotePlanRegistration {
+    #[structopt(short = "vs", long = "vote-start")]
+    pub vote_start: BlockDate,
+
+    #[structopt(short = "ve", long = "vote-end")]
+    pub vote_end: BlockDate,
+
+    #[structopt(short = "ce", long = "committee-end")]
+    pub committee_end: BlockDate,
+
+    // #[structopt(short = "p", long = "proposals")]
+    // pub proposals: Proposals,
```

Comment:
@eugene-babichenko , `Proposals` is a `Vec<Proposal>`, what would be the correct way of integrating this with `StructOpt`?

Reply:
Use `Vec<Proposal>` and then initialize `Proposals` with this vector.

Reply:
please refer to the "type magic" part for the details https://docs.rs/structopt/0.3.14/structopt/#type-magic

Reply:
create a `Proposal` type that can be read from the cli. for now we just need the `proposal id` and use default `VoteOption::new(0b0011)` for now

Reply:
Gonna do that @NicolasDP 
Thanks for the info anyway @eugene-babichenko!